src/index.js

```diff
@@ -57,6 +59,7 @@ export default function supernova(env) {
         senseNavigation,
         multiUserAutomation,
         translator,
+        isBackgroundLoaded,
```

Comment:
We should probably wrap `renderButton` in a `useEffect` and add `isBackgroundLoaded` to the dependency array. Then make sure `isBackgroundLoaded` is true, to run `renderButton`.

Reply:
Why do you think  it is needed ?
`isBackgroundLoaded ` is called when the url changes in the layout via `useLoadImage`. It keeps the button rendering until the image is loaded properly. I don't think I need to change anything else.

Reply:
Because it is kind of a backwards way of doing it IMO. You shouldn't really render anything until you actually have loaded the image. Then you don't have to pass a boolean all the way to styling logic

Reply:
`isBackgrounfLoaded` is a check for the case that the button gets background image, it can be true or false. When the button has no background image the url is undefined and we still need the button to be rendered and get the rest of the style.

Reply:
Then we can just always make isBackgroundLoaded to be true when there is no background

Reply:
I don't know which solution is correct. The bug is still there,  I could reproduce it however it happens less often. I'll try your idea for not passing the variable down into getStyles.